skyportal/tests/api/test_groups.py

```diff
@@ -245,37 +244,10 @@ def test_add_stream_to_single_user_group_delete_stream(
         data={"stream_id": public_stream.id},
         token=super_admin_token,
     )
-    assert status == 200
-    assert data["data"]["stream_id"] == public_stream.id
-
-    # check that stream is there
-    status, data = api(
-        "GET",
-        f"groups/{single_user_group['id']}",
-        token=super_admin_token,
-    )
-    assert data["data"]["streams"][0]["id"] == public_stream.id
-
-    # delete stream
-    status, data = api(
-        "DELETE",
-        f"streams/{public_stream.id}",
-        token=super_admin_token,
-    )
-    assert status == 200
-
-    # check it is deleted from group
-    status, data = api(
-        "GET",
-        f"groups/{single_user_group['id']}",
-        token=super_admin_token,
-    )
-    assert len(data["data"]["streams"]) == 0
 
-    # check user still exists
-    status, data = api("GET", f"user/{new_user_id}", token=super_admin_token)
-    assert status == 200
-    assert data["data"]["id"] == new_user_id
+    # check that you can't add a stream to a single user group
+    assert status == 400
+    assert data['status'] == 'error'
```

Comment:
Refactored the above as users should not be allowed to add stream requirements to single user groups